html/main_float.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Cubes</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../css/style_float.css">
+</head>
+<body>
+
+<div class="content">
```

Comment:
Either use the BEM methodology or give your classes meaningful names, now classes' names are very common. Also try to use semantic tags.